modules/GT_Hazards.php

```diff
@@ -68,7 +59,7 @@ function getHazardRoll($game, $card, $progress = null)
       'die1' => $die1,
       'die2' => $die2,
       'row_col' => $row_col,
-      'type' => $card['type'] == 'meteoric' ? 'meteor' : 'cannon',
+      'type' => $card->getType() == CARD_METEORIC_SWARM ? 'meteor' : 'cannon',
```

Comment:
Any desire to refactor these stings into constants right now? Or maybe that should be a separate refactor...

Reply:
There's too much for one task, I'd rather merge this now and transition to constants where needed later